base/Base.jl

```diff
@@ -593,14 +593,19 @@ in_sysimage(pkgid::PkgId) = pkgid in _sysimage_modules
 if is_primary_base_module
 
 # Profiling helper
-# triggers printing the report and (optionally) saving a heap snapshot after a SIGINFO/SIGUSR1 profile request
+# triggers printing the report and (optionally) saving a heap snapshot/printing task backtraces
+# after a SIGINFO/SIGUSR1 profile request
 # Needs to be in Base because Profile is no longer loaded on boot
 function profile_printing_listener(cond::Base.AsyncCondition)
     profile = nothing
     try
         while _trywait(cond)
             profile = @something(profile, require_stdlib(PkgId(UUID("9abbd945-dff8-562f-b5e8-e1ebf5ef1b79"), "Profile")))::Module
             invokelatest(profile.peek_report[])
+            if Base.get_bool_env("JULIA_PROFILE_PEEK_TASK_BACKTRACES", false) === true
```

Comment:
I think it is worth making this the default even
```suggestion
            if Base.get_bool_env("JULIA_PROFILE_PEEK_TASK_BACKTRACES", true) === true
```

It would also potentially be great to put this in `signals-unix.c : signal_listener` directly, so it operates on fatal signals too

Reply:
I've changed the PR so that we print the backtraces by default :+1: As for calling this from the signal handler directly -- do you mean we should do also print these for different signals than SIGUR1/SIGINFO? I think I'd need some guidance with that as I'm not sure which signals would be suitable to be included.

Reply:
FYI: at least for us, with our huge sysimg, this can be really slow. Like >5 minutes slow.

So if that impacts others too, do we want to do this by default?

Reply:
Why does system image size affect this? Yes, that would be the list of all "critical" signals defined there. There's already a place we call `jl_print_bt_entry_codeloc` on the threads, so it would just tack on right after that.

Reply:
We believe the reason is because symbolizing the traces is super slow with our giant binary size. But to be honest i think we don't know why it's so slow.

Reply:
Ah, gotcha, that does make sense. Symbolizing something big (such as the LLVM debug info when asserts are on) can take quite some time.

Reply:
Although that happens in parallel with the runtime, so I think that still might be okay, for something the user has to request pretty explicitly